<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomGameDlg</class>
 <widget class="QDialog" name="CustomGameDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="myGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>My pieces settings</string>
   </property>
   <widget class="QSpinBox" name="myKingSB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="myPiecesSB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="myKingLB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kings amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MyPieceLB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pieces amount:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="compGroupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>Computer pieces setting</string>
   </property>
   <widget class="QSpinBox" name="compKingSB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="compPiecesSB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="compKingLB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kings amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="compPieceLB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pieces amount:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okPB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
